FAQ.md

```diff
@@ -42,6 +42,24 @@ OAuth authentication requires the end-user to interact with a web browser, which
 
 It is possible to user OAuth authentication in a web application, but not using this `GoogleWebAuthorizationBroker` class.
 
+One way to do it is OAuth via JWT:
```

Comment:
Since this is not performing end user authentication it's really not an alternative to `GoogleWebAuthorizationBroker`.

Also the general purpose of this FAQ is not to provide documentation, tutorials. guide or examples on how to perform Auth using the Google.Apis.Auth.* libraries. Instead we just answer questions that have come up over and over.

There's ample general purpose documentation about [using OAuth 2.0 for accesing Google APIs](https://developers.google.com/identity/protocols/oauth2) and for using the [.NET Google.Apis.Auth.* libraries](https://developers.google.com/api-client-library/dotnet/guide/aaa_oauth). Service account authentication is covered in both, and with examples on the latter.

Reply:
Server side OAuth is for service accounts not end users.
Anyway this is a problem I had to solve and it felt like faq tells me what's the problem but does not offer a solution while ack. it exists. Including this would probably save some users a headache.

Reply:
> Server side OAuth is for service accounts not end users.

Precisely, so your example, which is still correct for service accounts OAuth, is not a good alternative to the question in this FAQ which is about `GoogleWebAuthorizationBroker` which is exclusively for end user OAuth. The true alternative, i.e. if you need to authentica end users in a web application, is to use one of out integration packages, like `Google.Apis.Auth.AspNetCore3` or build your own package.

Take a look at #2044, I've expanded the answer to the question, and also included the links to documentaion about OAuth and Google APIs, both general documentation and .NET specific. Docs cover all use cases, including service account OAuth and end user OAuth in both desktop and server applications.

Reply:
As a consumer of API I prefer to get direct answers instead of being redirected to docs (the point of me using a service is the service saves me time instead of working with a lower level lib) when possible. Now this is a wild guess but I would expect service account oauth to be a common use case and I still think readers would benefit from this. 

Maybe this could be included a separate entry in faq then?

Reply:
The documentation links I included are the official documentation for Google OAuth and Google OAuth integration with .NET. Service account auth is indeed a very common use case as well as all variants of end user auth, those are all covered on the official documentation (do please feel free to create issues if you feel there's something missing there). We are also working on making documentaiton more discoverable in general.

This page *is not* a documentation page where we describe general usage of standard features. And, in any case, we wouldn't duplicate documentation as that's an overhead for everyone, and will end up affecting users when duplicated docs become out of sync and stale. We feel adding the links on the FAQs page is a good compromise in the sense that it points users to the right documention without creating confusion, duplicating efforts and risking stale information. I'm sorry you don't agree.